spec/rubocop/cop/rspec/spec_file_path_format_spec.rb

```diff
@@ -233,19 +233,66 @@ class Foo
     RUBY
   end
 
-  context 'when configured with `CustomTransform: { "FooFoo" => "foofoo" }`' do
-    let(:cop_config) { { 'CustomTransform' => { 'FooFoo' => 'foofoo' } } }
+  context 'when configured with `CustomTransform: { "RSpec" => "rspec" }`' do
+    let(:cop_config) do
+      {
+        'CustomTransform' => { 'RSpec' => 'rspec' },
+        'CustomTransformPatterns' => {}
+      }
+    end
+
+    it 'registers an offense when not an exact match to custom ' \
+       'module name transformation' do
+      expect_offense(<<-RUBY, 'rspec_foo_spec.rb')
+        RSpec.describe RSpecFoo do; end
```

Comment:
The best way to demo the difference between those two option would be `RSpec.describe CustomDSL` and `DSL` => `dsl` mapping. One would insist on `custom_d_s_l_*spec.rb`, the other on `custom_dsl_*spec.rb`, right?

Namespaces and `RSpecFoo` don't help understand it.

I keep losing grip about this option.

Reply:
This does not appear to be an offense when followed by only one uppercase letter, as in DSL, so this is not the case that can be resolved with this option. This option may not be a common use case.